src/java.base/share/classes/jdk/internal/foreign/SystemLookup.java

```diff
@@ -80,16 +85,41 @@ private static SymbolLookup makeSystemLookup() {
         }
     }
 
+    /* Obtain the address of the inlined function list by loading the specified library. */
+    private static MemorySegment getInlinedFunctListAddr() {
+         MemorySegment funcsAddr = null;
```

Comment:
Please remove the unnecessary initializer.

Reply:
Fixed.